Makefile

```diff
@@ -11,5 +11,4 @@ test-integ: test
 
 version ?= latest
 test-docker:
-	curl -s https://raw.githubusercontent.com/sendgrid/sendgrid-oai/HEAD/prism/prism.sh -o prism.sh
-	version=$(version) bash ./prism.sh
+	docker compose up --build --parallel --force-recreate --abort-on-container-exit --remove-orphans
```

Comment:
docker-compose.yml file is present in `sendgrid/sendgrid-oai` project.

Reply:
Ouch I see. But that project is archived. Is there any newer alternative or something?